sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -570,6 +570,26 @@ export interface GenerateSharedAccessSignatureOptions {
     apiVersion?: string;
 }
 
+// @public
+export type HealthcareFhirResourceCreatedEventData = HealthcareFhirResourceEventBase & {};
+
+// @public
+export type HealthcareFhirResourceDeletedEventData = HealthcareFhirResourceEventBase & {};
+
+// @public
+export interface HealthcareFhirResourceEventBase {
+    resourceFhirAccount: string;
+    resourceFhirId: string;
+    resourceType: HealthcareFhirResourceType;
+    resourceVersionId: number;
+}
+
+// @public
```

Comment:
I forget why this type alias is useful, is it for the doc string on it with known values?